### PR TITLE
Memory leaks

### PR DESCRIPTION
**Pull request by jxy**

### CODE REVIEW

File: src/cmd/acme/rows.c [hunk 1]
```diff
@@ -627,6 +628,7 @@ rowload(Row *row, char *file, int initing)
 			textdelete(&row->tag, 0, row->tag.file->b.nc, TRUE);
 			textinsert(&row->tag, 0, r, nr, TRUE);
 			break;
+			free(r);
```
Comment (fhs) [inline src/cmd/acme/rows.c]: unreachable line

Comment (jxy) [reply src/cmd/acme/rows.c]: nice catch

File: src/libdraw/openfont.c [hunk 2]
```diff
@@ -42,19 +42,21 @@ openfont1(Display *d, char *name)
 		nambuf = smprint("#9/font/%s", fname+14);
 		if(nambuf == nil)
 			return 0;
-		nambuf = unsharp(nambuf);
+		char *nambuf0 = unsharp(nambuf);
```
Comment (0intro) [inline src/libdraw/openfont.c]: Please declare nambuf0 at the beginning of the function.

**Comment by 0intro** [comment]
Thanks for working on this.

Could you squash "acme: fix some memory leaks" and "acme: rowload: free then break" in a single commit?

**Comment by 0intro** [comment]
Merged. Thanks.

**Pull request merged**